Resource/template/admin/index.twig

```diff
@@ -80,145 +83,194 @@ function fnChangeActionSubmit(action) {
-                                    <div class="form-group">{{ form_widget(searchForm.recommend_level) }}</div>
-                                </div>
-                            </div>
-                            <div class="row">
-                                <div class="col-sm-6">
-                                    <label>{{ searchForm.review_start.vars.label }}</label>
-                                    <div class="form-group range">
-                                        {{ form_widget(searchForm.review_start, {'attr': {'class': 'input_cal'}}) }} ～ {{ form_widget(searchForm.review_end, {'attr': {'class': 'input_cal'}}) }}
-                                    </div>
-                                </div>
-                                <div class="col-sm-6">
-                                    <label>{{ searchForm.status.vars.label }}</label>
-                                    <div class="form-group">{{ form_widget(searchForm.status) }}</div>
-                                </div>
-                            </div><!-- /.row -->
-                            <div class="col-sm-12">
-                                <p class="text-center"><a href="#" class="search-clear">検索条件をクリア</a></p>
-                            </div>
-                        </div>
+        <div class="c-outsideBlock__contents">
+            <div class="row justify-content-start">
+                <div class="col-6">
+                    <div class="mb-2">
+                        <label class="col-form-label">{{ 'admin.product.index.470'|trans }}</label>
+                        {{ form_widget(searchForm.multi, { attr: { placeholder: 'plugin.admin.product_review.search.inputsearch.placeholder'|trans, class : 'input_search' } } ) }}
+                    </div>
+                    <div class="d-inline-block mb-3" data-toggle="collapse" href="#searchDetail"
+                         aria-expanded="false" aria-controls="searchDetail"><a><i
+                                    class="fa fa-plus-square-o font-weight-bold mr-1"></i><span
+                                    class="font-weight-bold">{{ 'admin.product.index.detailed_search'|trans }}</span></a>
                     </div>
                 </div>
-            </div><!-- /.col -->
+            </div>
         </div>
-        <div class="row btn_area">
-            <div class="col-xs-8 col-xs-offset-2 col-sm-4 col-sm-offset-4 text-center">
-                <button class="btn btn-primary btn-block btn-lg" onclick="fnChangeActionSubmit('{{ url('plugin_admin_product_review') }}'); return false;">
-                    検索する <svg class="cb cb-angle-right"><use xlink:href="#cb-angle-right"></use></svg>
-                </button>
-            </div> <!-- /.col -->
+        <div class="c-subContents collapse ec-collapse" id="searchDetail">
+            <div class="row mb-2">
+                <div class="col-6 col-sm-4">
+                    <label>{{ searchForm.product_name.vars.label }}</label>
```

Comment:
Should have `trans` to translate label 
```
 searchForm.product_name.vars.label|trans
```

Reply:
This code is used to get the label of the field